.gitmodules

```diff
@@ -0,0 +1,13 @@
+
+[submodule "thirdparty/cpu_features"]
+	path = thirdparty/cpu_features
+	url = https://github.com/google/cpu_features.git
+[submodule "thirdparty/nlohmann_json"]
```

Comment:
why change nlohmann_json and easylogging to submodule and download them every time ?

Reply:
submodule is normal way in opensource project.